### PR TITLE
added header w.logotype

### PR DESCRIPTION
**Pull request by mvic89**

### CODE REVIEW

File: index.html
Comment (sabawaheed27) [inline index.html:1]: good job

File: Gitguide-assignment
Comment (bushra-rauf) [inline Gitguide-assignment:1]: looks good

File: project.html
Comment (bushra-rauf) [inline project.html:1]: ok

File: styles/global.css
Comment (bushra-rauf) [inline styles/global.css:1]: okay

**Review by bushra-rauf — APPROVED** [review]

File: project.html
Comment (sabawaheed27) [inline project.html:1]: can you change footer name, it should be team name

Comment (sabawaheed27) [inline project.html:1]: why about us is on top ?

**Review by bushra-rauf — COMMENTED** [review]
looks good

**Pull request merged**